### PR TITLE
Be able to run Uyuni master CI in Provo

### PR DESCRIPTION
**Pull request by Bischoff**
Motivated by datacenter migration

### CODE REVIEW

File: terracumber_config/tf_files/Uyuni-Master-PRV.tf [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  use_avahi    = false
+  name_prefix  = "uyuni-master-"
+  domain       = "mgr.prv.suse.net"
+  from_email   = "root@suse.de"
+
+  no_auth_registry = "registry.mgr.prv.suse.net"
+  auth_registry      = "registry.mgr.prv.suse.net:5000/cucutest"
+  auth_registry_username = "cucutest"
+  auth_registry_password = "cucusecret"
+  git_profiles_repo      = "https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/internal_nue"
```
Comment (srbarrios) [inline terracumber_config/tf_files/Uyuni-Master-PRV.tf:116]:
```suggestion
  git_profiles_repo      = "https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/internal_prv"
```

File: jenkins_pipelines/environments/uyuni-master-dev-acceptance-tests-PRV [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env groovy
+
+node('sumaform-cucumber') {
```
[the remark this answers is not in the document]
Comment (Bischoff) [reply jenkins_pipelines/environments/uyuni-master-dev-acceptance-tests-PRV:3]: Ouch!!! thanks!

File: terracumber_config/tf_files/Uyuni-Master-PRV.tf (hunk 1 above)
Comment (Bischoff) [reply terracumber_config/tf_files/Uyuni-Master-PRV.tf]: Fixed

File: jenkins_pipelines/environments/uyuni-master-dev-acceptance-tests-PRV (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bischoff) [reply jenkins_pipelines/environments/uyuni-master-dev-acceptance-tests-PRV]: Fixed

**Review by srbarrios — APPROVED** [review]

**Pull request merged**